blog/2021-09-17-announcing-stryker-net-1.md

```diff
@@ -0,0 +1,265 @@
+---
+slug: 2021-09-17-announcing-stryker-net-1.md
+title: Announcing Stryker.NET 1.0
+author: Richard Werkman
+author_title: Stryker Team
+author_url: https://github.com/richardwerkman
+author_image_url: https://avatars.githubusercontent.com/u/14224500?v=4
+tags: [stryker.net]
+---
+
+# Announcing Stryker.NET 1.0 - No more beta!
+
+We're proud to announce the first major release of Stryker.NET: 1.0. It comes with exciting new features and a overhaul of how options work. It should now be easier to use.
+
+<!--truncate-->
+
+If you're new to mutation testing, it's a way to measure your tests' effectiveness. A mutation testing framework will make small changes, called _mutants_, one by one in your source code. Then it will run your tests to see if one of them fails. If so, you just "killed" that mutant; if not, it "survived". If too many mutants survive, you probably want to improve your tests. The mutation testing report will give you insides into the test cases you may have missed. If this all sounds complicated, please take a look at [our RoboBar 🤖🍷 example](https://stryker-mutator.io/example).
+
+If you're new to Stryker.NET, please follow our [Getting started guide](https://stryker-mutator.io/docs/stryker-net/getting-started/). Are you already using Stryker.NET? Update to the latest version with the following command:
+
+```shell
+dotnet tool update -g dotnet-stryker
+```
+
+With that out of the way, let's dive into the new stuff!
+
+## 💥 Breaking changes
+
+### ⏭ Updated runtime
+
+The .NET runtime for Stryker.NET has been updated from 3.1 to 5.0.
+
+This should have some performance benefits. And helps with developing Stryker.NET in the future.
+
+Please [download and install .NET 5.0](https://dotnet.microsoft.com/download/dotnet/5.0) or update your pipeline to support .NET 5.
+
+### ⏭ Options rework
+
+Almost all options have been renamed or work different. A fundamental difference on the CLI is how multi value options are passed.
+
+#### Multi value options
+
+The old annotation for passing multi value options was confusing and not based on any standards. For example this is how multiple reporters were passed 👎
+
+```shell
+dotnet stryker --reporters "['html', 'progress']"
+```
+
+This now looks like 👍
+
+```shell
+dotnet stryker --reporter "html" --reporter "progress"
+```
+
+#### Options migration guide
+
+A lot of options have been renamed or have been removed from the CLI. They can now only be passed in the JSON. Other options are now only available trough CLI. For example a API key should not be stored in JSON so that possibility has been removed.
```

Comment:
```suggestion
A lot of options have been renamed. We have also decided that some options either do not belong on the commandline or don't belong in the configuration file. For example an API key should not be stored in the configuration file so that possibility has been removed.
```